Codecademy/gamut
# fix: bookmarks and notification popovers esc key a11y

### Overview


Fix made for a11y issue with notifications and bookmarks where pressing `esc` (or clicking outside) did not close the desktop popovers.
Deprecated renderBookmarks has been removed.




### PR Checklist

- [x] Related to designs: https://www.figma.com/file/5rkfS3uWZsFNTsx2GmlDYA/Save-Feature-MVP
- [x] Related to JIRA tickets: [REACH-2032](https://codecademy.atlassian.net/browse/REACH-2032) & [REACH-1977](https://codecademy.atlassian.net/browse/REACH-1977)

#### Testing Instructions

Must be tested within portal-app or monolith. See testing instructions in linked PRs for each.

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/32179)  | [Monolith Env](https://pr-32179-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2450)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2450)   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |
